x-pack/packages/observability/alerting_test_data/src/scenarios/custom_threshold_log_count_groupby.ts

```diff
@@ -22,7 +23,7 @@ export const scenario2 = {
     params: {
       criteria: [
         {
-          aggType: Aggregators.CUSTOM,
+          aggType: CUSTOM_AGGREGATOR,
```

Comment:
I changed it to `CUSTOM_AGGREGATOR` because it is the only acceptable value at this level and does not belong to `Aggregators` anymore. There were a lot of type duplications to handle this scenario, and by splitting them, we don't need to exclude `custom` from `Aggregators` to be used in the rule.

I am considering creating a ticket to remove `CUSTOM_AGGREGATOR` everywhere if it does not have a backward compatibility issue.